src/views/GameView.vue

```diff
@@ -4,17 +4,14 @@
       <div class="col">
         Lobby Name: <input v-model="lobbyName" /> Player Name:
         <input v-model="playerName" />
-        <button class="btn btn-sm btn-info" @click="connectGame">
-          Create game
+        <button class="btn btn-sm btn-info" @click="connectToLobby">
+          Create/Join Lobby
         </button>
-        <button class="btn btn-sm btn-success" @click="startTaskGame">
-          Start game
+        <button class="btn btn-sm btn-success" @click="startLobby">
```

Comment:
createLobby / create Lobby?

Reply:
no this button (not implemented yet) starts the actuall game when everyone is already in the lobby, becouse i want to be consistant not its confusing useing at some points lobby and in some points game i just decidet to use everywhere lobby and make it more obvious what is ment with an additional attribute.